src/Package/Impl/DataInspect/VariableView.xaml.cs

```diff
@@ -71,6 +68,16 @@ public partial class VariableView : UserControl, ICommandTarget, IDisposable {
             _environmentProvider.RefreshEnvironmentsAsync().DoNotWait();
         }
 
+        private void OnUIThemeChanged(object sender, EventArgs e) {
+            SetImageBackground();
+        }
+
+        private void SetImageBackground() {
```

Comment:
Yes, but you can encapsulate it inside `AttachedProperty` and set it in ctor of the control like:
```cs
UITheme.SetShell(this, _shell);
```

This is better than adding a base class because we may need more than one base class for it